k8s/upgrades/README.md

```diff
@@ -16,7 +16,7 @@ If already using CSPC pools and cStor CSI volumes they can be upgraded from `1.1
 
 This document describes the steps for the following OpenEBS Upgrade paths:
 
-- Upgrade from 1.0.0 or later to a newer release up to 2.0.0
+- Upgrade from 1.0.0 or later to a newer release up to 2.1.0
```

Comment:
question: Should we support from 1.0.0 to 2.1.0 directly. even though it works.
May be 1.0.0 to 2.0.0, and then 2.0.0 to 2.1.0?

I think last release we had discussed about splitting the upgrade document to reflect these changes.

Reply:
This is for the spc and older provisioned volumes so still we support 1.0.0-2.1.0. 
Once we completely move this doc to another repo and make cspc and csi volumes as the main README for upgrade we need to update it.